src/Formats/ProtobufSchemas.h

```diff
@@ -21,28 +25,28 @@ namespace protobuf
 namespace DB
 {
 class FormatSchemaInfo;
-/// proton: starts
-struct SchemaValidationError;
-using SchemaValidationErrors = std::vector<SchemaValidationError>;
-/// proton: ends
 
 /** Keeps parsed google protobuf schemas parsed from files.
   * This class is used to handle the "Protobuf" input/output formats.
   */
-class ProtobufSchemas : private boost::noncopyable
+class ProtobufSchemas : private boost::noncopyable, public google::protobuf::io::ErrorCollector /* proton: updated */
```

Comment:
We can remove `private boost::noncopyable` probably since the data member std::mutex is not copyable which effectively make ProtobufSchemas non-copyable.